package.json

```diff
@@ -30,5 +32,10 @@
     "url": "https://github.com/parasoft/parasoft-findings-vsts/issues"
   },
   "homepage": "https://github.com/parasoft/parasoft-findings-vsts#readme",
-  "icon": "images/icon.png"
+  "icon": "images/icon.png",
+  "dependencies": {
+    "@types/jasmine": "^4.3.1",
+    "del-cli": "^5.0.0",
```

Comment:
This dependency might be an overkill for just cleaning files after tests.
Consider to do it with JavaScript in places like "afterEach".
I researched a little bit, maybe consider to read directory with following API:
https://nodejs.org/api/fs.html#fsreaddirpath-options-callback
And delete files with following API:
https://nodejs.org/api/fs.html#fsunlinkpath-callback